### PR TITLE
cloudtests: testdrive: allow configuring default URLs 

### PR DESCRIPTION
**Pull request by nrainer-materialize**
### Motivation

This allows configuring the default URLs for testdrive.

### CODE REVIEW

**Comment by nrainer-materialize** [comment]
Simplified the code, I don't see a reason why we would need to allow overriding the parameters again in the `run` method.

**Review by philip-stoev — APPROVED** [review]
Thank you

File: misc/python/materialize/cloudtest/k8s/testdrive.py [hunk 1]
```diff
@@ -23,9 +23,17 @@ def __init__(
         release_mode: bool,
         aws_region: Optional[str] = None,
         namespace: str = DEFAULT_K8S_NAMESPACE,
+        materialize_url: str = "postgres://materialize:materialize@environmentd:6875/materialize",
+        materialize_internal_url: str = "postgres://materialize:materialize@environmentd:6877/materialize",
```
Comment (philip-stoev) [inline misc/python/materialize/cloudtest/k8s/testdrive.py:27]: I think this default is wrong, it should be mz_system without a password . Not sure how it even works right now.

Comment (nrainer-materialize) [reply misc/python/materialize/cloudtest/k8s/testdrive.py:27]: Indeed. This was already there, is it nowhere used? 😱

**Pull request merged**